apps/examples/reboot_reason_test/Kconfig

```diff
@@ -0,0 +1,17 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_REBOOT_REASON
+	bool "\"Reboot Reason\" example"
+	default n
+	depends on SYSTEM_REBOOT_REASON
+	select MM_ASSERT_ON_FAIL
+	select WATCHDOG
+	---help---
+		Enable the \"Reboot Reason\" example
```

Comment:
How about below for help?
TizenRT provides the functionality to know the reason of previous rebooting. This example gives several options to check it.